app.py

```diff
@@ -15,6 +15,8 @@
 
 VALID_CAMPUSES = ["Daytona Beach", "Cocoa", "Sanford-Lake Mary", "Lake Nona", "Downtown", "Rosen"]
 
+#Hello world
```

Comment:
I don't think that this is a valid comment to describe TODO. Please remove and say "Hi World" 

@pieguy40 please review 
![download](https://user-images.githubusercontent.com/29994360/105643728-93d44c00-5e5f-11eb-8779-b8be3aec2253.jpg)